test/extended/ldap.go

```diff
@@ -0,0 +1,75 @@
+// +build ldap
```

Comment:
This test is (currently) testing the fixtures, and will in the future be expanded to also test the group sync - why `extended/authentication` ?